src/content/developers/docs/consensus-protocols/pow/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Proof of work (PoW)
+description: An explanation of the proof of work consensus protocol and its role in Ethereum.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+Ethereum, like Bitcoin, currently uses a consensus protocol called [Proof of Work (PoW)](https://en.wikipedia.org/wiki/Proof_of_work) consensus protocol. This allows the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain, and prevents certain kinds of economic attacks.
+
+## Prerequisites {#prerequisites}
+
+To better understand this page, we recommend you first read up on [transactions](/en/developers/docs/transactions/) and [blocks](/en/developers/docs/blocks/).
+
+## What is Proof of Work (PoW)?
+
+In decentralized systems like Ethereum, we need to ensure that everyone agrees on the order of transactions. The Proof of Work consensus mechanism allows Ethereum to achieve this.
+
+The Proof of Work consensus mechanism is currently the most widely-used consensus mechanism. It was pioneered by Satoshi Nakamoto with the release of Bitcoin in 2008 and it has powered the majority of high-profile blockchains, including Ethereum.
+
+## How does it work?
+
+<!-- TODO how does it work -->
+
+## What is mining?
+
+Mining is the lifeblood of Proof of Work. It is the process of creating a block transactions to be added to the Ethereum blockchain.
```

Comment:
Feel like the second sentence might have some typos. Should it be "block of transactions"?